src/main/java/de/uhd/ifi/se/decision/management/jira/extraction/view/macros/IssueMacro.java

```diff
@@ -45,9 +55,38 @@ public static String reformatCommentBody(String inputBody) {
 		}
 		return body;
 	}
-	
+
+	/**
+	 * Static function for other Macro Classes
+	 * 
+	 * @param renderContext
+	 * @return
+	 */
 	public static String getProjectKey(RenderContext renderContext) {
 		return renderContext.getParams().get("jira.issue").toString().split("-")[0];
 	}
 
+	/**
+	 * Static function for other Macro Classes
+	 * 
+	 * @param renderContext
+	 * @param body
+	 * @param type
+	 * @return the js context menu call for comment tab panel
+	 */
+	public static String getContextMenuCall(RenderContext renderContext, String body, String type) {
+		long id = 0;
+		if (renderContext.getParams().get("jira.issue") instanceof IssueImpl) {
```

Comment:
Is it possible to get the comment id from the render context parameters? I think with the comment id, you wouldn't have to make the string matching for all sentences for a JIRA issue.
Instead of calling the method "getElementsForIssueWithType" it would be better to call a method "getElementsForCommentWithType(long commentId, ...)"